include/libdnf/comps/group/group.hpp

```diff
@@ -131,10 +125,10 @@ class Group {
     void dump(const std::string & path);
 
 protected:
-    explicit Group(GroupQuery * query);
+    explicit Group(libdnf::Base & base);
 
 private:
-    GroupQueryWeakPtr query;
+    libdnf::BaseWeakPtr base;
```

Comment:
I went with weak pointer to base, but I am not sure if it's the right choice, because I can also imagine comps or sack could be used.